src/gh.js

```diff
@@ -21,14 +21,9 @@ async function getRunner(label) {
 async function getRegistrationToken() {
   const octokit = github.getOctokit(config.input.githubToken);
 
-  try {
-    const response = await octokit.request('POST /repos/{owner}/{repo}/actions/runners/registration-token', config.githubContext);
-    core.info('GitHub Registration Token is received');
-    return response.data.token;
-  } catch (error) {
-    // core.error('GitHub Registration Token receiving error');
-    throw error;
-  }
+  const response = await octokit.request('POST /repos/{owner}/{repo}/actions/runners/registration-token', config.githubContext);
```

Comment:
These changes are to satsify the linter. 

This try/catch block doesn't do anything in catch besides throwing the error, so it is effectively not doing anything. Removing the try/catch black will have the same outcome as the previous code.